DependencyInjection/Configuration.php

```diff
@@ -215,6 +215,8 @@ private function addFilesystemSection(ArrayNodeDefinition $node)
                                     ->end()
                                 ->end()
                                 ->scalarNode('region')->defaultValue('s3.amazonaws.com')->end()
+                                ->scalarNode('version')->defaultValue('latest')->end()
+                                ->scalarNode('sdk_version')->defaultValue(2)->end()
```

Comment:
Can you limit the possible values?

http://symfony.com/doc/current/components/config/definition.html#enum-nodes